src/VirtualMachine/Ddev/Client.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace UnleashedTech\DeployerRecipes\VirtualMachine\Ddev;
+
+use UnleashedTech\DeployerRecipes\VirtualMachine\AbstractClient;
+
+use function Deployer\runLocally;
+
+class Client extends AbstractClient
+{
+    public function run(string $command): void
```

Comment:
Similar to the return value used in the parent run function, I would recommend the simple change of the void to string here.  As seen in this MR https://github.com/unleashedtech/deployer-recipes/pull/15/files#diff-b2b35efa92e9f5512fa3982fa2db7d9857f08ab94437448afe3e0eaedd7efa8fR21  .  And then we can use it like so:   https://github.com/unleashedtech/deployer-recipes/blob/f794c5ee6b56e69b0f75f30e6473f70fda768ebc/cms/drupal/db/backup/import.php#L23

Reply:
Nice! Will implement!